app/pdfs/response_report.rb

```diff
@@ -46,7 +46,8 @@ def header_text(str, size: 24)
   end
 
   def creation_date
-    text "Ingevuld op #{I18n.l @response.created_at, format: :long}", style: :bold_italic
+    text "Aangevraagd op: #{I18n.l @response.requested_at, format: :long}", style: :italic
```

Comment:
Die data leken me juist wel belangrijk, het is (op het moment) de enige manier voor een behandelaar om een resultaat aan een patiënt te koppelen.